plugins/code_commit/setup.py

```diff
@@ -22,10 +22,27 @@
 setup(
     name="aws-orbit-code-commit",
     version=version,
+    author="AWS Professional Services",
+    author_email="aws-proserve-opensource@amazon.com",
+    url="https://github.com/awslabs/aws-orbit-workbench",
+    project_urls={"Org Site": "https://aws.amazon.com/professional-services/"},
     description="Orbit Workbench CodeCommit Plugin.",
     license="Apache License 2.0",
     packages=["code_commit"],
+    keywords=["aws", "cdk"],
     python_requires=">=3.6, <3.9",
+    classifiers=[
+        "Development Status :: 4 - Beta",
+        "Framework :: Jupyter",
+        "Intended Audience :: Developers",
+        "Intended Audience :: Information Technology",
+        "Intended Audience :: Science/Research",
+        "License :: OSI Approved :: Apache Software License",
+        "Programming Language :: Python :: 3 :: Only",
+        "Programming Language :: Python :: 3.7",
+        "Programming Language :: Python :: 3.8",
+        "Typing :: Typed",
+    ],
     install_requires=[
         "aws-cdk.aws-codecommit~=1.67.0",
```

Comment:
we should drop these cdk references from the install_requires as they'll be installed by our codebuild